saleor/graphql/order/mutations/orders.py

```diff
@@ -671,3 +672,42 @@ def perform_mutation(cls, _root, info, **data):
         instance.status = InvoiceStatus.READY
         instance.save()
         return UpdateInvoice(invoice=instance)
+
+
+class SendInvoiceEmail(ModelMutation):
+    class Arguments:
+        id = graphene.ID(required=True, description="ID of an invoice to be sent.")
+
+    class Meta:
+        description = "Send an invoice by email."
+        model = models.Invoice
+        permissions = (OrderPermissions.MANAGE_ORDERS,)
+        error_type_class = InvoiceError
+        error_type_field = "invoice_errors"
+
+    @classmethod
+    def clean_instance(cls, info, instance):
+        validation_errors = {}
+        if instance.status != InvoiceStatus.READY:
+            validation_errors["invoice"] = ValidationError(
```

Comment:
Here you're using `"invoice"` as a field for the validation error, but we use these fields to indicate which input field causes the error. In the case of this mutation, the validation error isn't really related to any input field (there is only `id`), so I would leave it empty and just return a generic error.  But then we would have to also change the generic `InvoiceErrorCode.INVALID` to something more specific like e.g `NOT_READY`.